### DISCUSSION TITLE
@types/splitting

### DISCUSSION THREAD

**Discussion by craigrileyuk** [DefinitelyTyped/DefinitelyTyped#63983] [Request a new @types package]
https://github.com/shshaw/splitting/

https://splitting.js.org/

https://www.npmjs.com/package/splitting

**Comment by MysteryBlokHed** [comment]
I just opened #64017

**Reply by craigrileyuk** [reply]
Awesome. Thank you!

**Reply by MysteryBlokHed** [reply]
PR has been merged 😄
